crashes/office-dashboard.js

```diff
@@ -0,0 +1,138 @@
+var gWidth;
+var gHeight;
+
+function graph_error(target, err) {
+  MG.data_graphic({
+    width: gWidth,
+    height: gHeight,
+    target: target,
+    chart_type: "missing-data",
+    error: err,
+    missing_text: "Failed to load data.",
+  });
+}
+
+document.addEventListener("DOMContentLoaded", function() {
+  var s = d3.select("#beta-graph");
+  gWidth = parseInt(s.style("width"));
+  gHeight = parseInt(s.style("height"));
+
+  get_beta_bydate().then(
+    function(data) {
+      setup_channel_graph(data, "beta");
+    },
+    graph_error.bind(undefined, "#beta-graph")
```

Comment:
(*nit*) bind with `null` is easier to comprehend (vs `undefined` as a way to actually check for undefined variables)